src/components/Loader/SunLoader.tsx

```diff
@@ -0,0 +1,50 @@
+
+import Typography from '../Typography';
+import React, { useState } from 'react';
+import './SunLoader.scss'
+
+export interface SunLoaderProps {
+    loadingText?: string[]
+}
+
+const loadingTexts = [
+    'Unblurring the lines ...',
+    'Estimating support ...',
+    'Considering credibility ...',
+    'Bursting filter bubbles ...',
+    'Unblurring the lines ...',
+  ];
+
+const SunLoader = (props: SunLoaderProps) =>
+{ 
+    const [loadingText, setLoadingText] = React.useState(
```

Comment:
We should use state from stories and since `useState` is defined in i`mport React, { useState } from 'react';` you can remove `React` here and leave just `useState`.

And one more thing, maybe pretty small one and I know that you said you are not too familiar with Typescript, but I think would be good to add type to every value, when we use `useState`. So in here I believe would be:
`const [loadingText, setLoadingText] = useState<string>(  'Unblurring the lines ...');` it would help us to avoid errors, bugs and so on

Reply:
I don't think we have to set the type necessarily. If you set `useState( 'Unblurring the lines ...')` then Typescript will know that is is a string.
If the type isn't clear it will give a warning (implicit 'any' type), then you have to set it.